README.md

```diff
@@ -92,14 +92,20 @@ scrape_configs:
         target_label: instance
       - target_label: __address__
         replacement: <<REDIS-EXPORTER-HOSTNAME>>:9121
+  ## it's also necessary
+  ## config for scraping the exporter itself
+  - job_name: 'redis_exporter'
+    static_configs:
+      - targets:
+        - <<REDIS-EXPORTER-HOSTNAME>>:9121
 ```
 
 The `targets-redis-instances.json` should look something like this:
 
 ```json
 [
   {
-    "targets": [ "redis-host-01:6379", "redis-host-02:6379"],
+    "targets": [ "redis://redis-host-ip-01:6379", "redis://redis-host-ip-02:6379"],
```

Comment:
Two things?
- first, what was the actual fix? using the `redis://` prefix? It works without that being present so not sure what's going on here
- you changed the name to use the `-ip` suffix but it doesn't need to be the IP, it can be the DNS hostname so I'd prefer to leave it like that to not make people think the actual IP is required.

Reply:
Thank you for your time and patience. 
After resolving the domain name, I try it again.
```
#### these are three examples work well.	
	"targets": [ "redis://127.0.0.1:22222", "redis://127.0.0.1:55555"],
    "targets": [ "redis-dns-name.com:22222", "redis-dns-name.com:55555"],
    "targets": [ "redis://redis-dns-name.com:22222", "redis://redis-dns-name.com:55555"],

#### but the follow example, It can't work
    "targets": [ "127.0.0.1:22222", "127.0.0.1:55555"],

#### error like this	
[root@TEST02 redis]# curl  http://10.1.15.217:9121/scrape?target="127.0.0.1:22222"
Invalid 'target' parameter, parse err: parse 127.0.0.1:22222: first path segment in URL cannot contain colon
```
Thank you very much

Reply:
I think your curl is wrong because of the quotes, try `curl  "http://10.1.15.217:9121/scrape?target=127.0.0.1:22222"`